example/instrum/Makefile.am

```diff
@@ -16,5 +16,19 @@ AM_LDFLAGS = -L$(PAPI_PATH)/lib -lpapi
 
 lib_LTLIBRARIES = $(LIB)/libinstrum.la
 
+noinst_HEADERS = \
+		$(srcdir)/instrum_common.h \
+		$(srcdir)/sample.h \
+		$(srcdir)/store.h \
+		$(srcdir)/papi_cnt.h \
+		$(srcdir)/init.h \
+		$(srcdir)/drv.h \
+		$(srcdir)/sched.h
```

Comment:
Are these headers used by anybody else? If not, you can just add them to _SOURCES. Also there is no need to specify $(srcdir) here.

Reply:
I kind of expected to have *.h in noinst_HEADERS and *.c in _SOURCES.